srcpkgs/lftp/template

```diff
@@ -10,10 +10,10 @@ hostmakedepends="pkg-config"
 makedepends="zlib-devel ncurses-devel readline-devel libressl-devel
  expat-devel libidn-devel gettext-devel"
 short_desc="Sophisticated FTP/HTTP client"
-maintainer="Nathan Owens <ndowens04@gmail.com>"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
Did @ndowens request or authorize this?

Reply:
No, this is due to: https://github.com/void-linux/void-packages/pull/24853#discussion_r487464613

Reply:
Ok, thanks